scripts/lint

```diff
@@ -1,12 +1,26 @@
 #!/usr/bin/env sh
 
+set -eu
+set -o pipefail
+
+printf 'linting CocoaPods\n'
+
+if ! command -v bundle > /dev/null; then
+  printf 'bundler is not installed.\n'
+  printf 'install with $ gem install bundler\n'
+  exit 1
+fi
+
+bundle config set path 'vendor/bundle'
+bundle install 
```

Comment:
can't we rely on the bootstrap script to install the gems?

Reply:
Can we rely on users to do the right things in the right order? I'd prefer to at least yell at them if it's not installed, and tell them to go run bootstrap.

Reply:
yes, let's do that.  making 'lint' time-consuming will encourage people to not use lint